AABB_tree/include/CGAL/AABB_traits_construct_by_sorting.h

```diff
@@ -0,0 +1,128 @@
+#ifndef AABB_TREE_TESTS_AABB_TRAITS_CONSTRUCT_BY_SORTING_H
```

Comment:
missing header

Reply:
I added a header with the same licensing as other headers in the package. Regarding crediting, should I list the same authors as `AABB_traits` since the new class was adapted from their code?

Reply:
yep